src/toggle.jsx

```diff
@@ -79,6 +79,7 @@ let Toggle = React.createClass({
       },
       label: {
         color: this.props.disabled ? this.getTheme().labelDisabledColor : this.getTheme().labelColor,
+        width: 'calc(100% - 46px)',
```

Comment:
I understand that you're allotting 46px of space to the toggle element, and the rest of the width to the label element. Is that correct? If so, is assigning 46px to the toggle always guaranteed to work?